### PR TITLE
Condor submission

### PR DESCRIPTION
**Pull request by gmauro**
enable Galaxy to send Condor parameters to Pulsar; add resource limits per destination and rename destinations to avoid duplication

### CODE REVIEW

File: files/galaxy-test/dynamic_rules/usegalaxy/destination_specifications.yaml [hunk 1]
```diff
@@ -104,42 +110,80 @@ sge:
...
+remote_condor_cluster_mq_it01:
+  limits:
+    cores: 16
+    mem: 31
   env: {}
   params:
     priority: -{PRIORITY}
-    request_cpus: '{PARALLELISATION}'
-    request_memory: '{MEMORY}'
+    submit_submit_request_cpus: '{PARALLELISATION}'
```
Comment (bgruening) [inline files/galaxy-test/dynamic_rules/usegalaxy/destination_specifications.yaml]: wow, why is this submit_submit_?

File: files/galaxy-test/dynamic_rules/usegalaxy/sorting_hat.py [hunk 2]
```diff
@@ -125,20 +127,27 @@ def build_spec(tool_spec, runner_hint=None):
     # semi-internal, and may not be properly propagated to the end tool
     tool_memory = tool_spec.get('mem', 4)
     tool_cores = tool_spec.get('cores', 1)
+    tool_gpus = tool_spec.get('cores', 0)
```
Comment (bgruening) [inline files/galaxy-test/dynamic_rules/usegalaxy/sorting_hat.py]: should this be `gpus`

File: files/galaxy-test/dynamic_rules/usegalaxy/sorting_hat.py [hunk 3]
```diff
@@ -161,6 +170,17 @@ def build_spec(tool_spec, runner_hint=None):
         if 'rank' in tool_spec:
             params['rank'] = tool_spec['rank']
 
+    if 'remote_condor_cluster_mq' in destination:
```
Comment (bgruening) [inline files/galaxy-test/dynamic_rules/usegalaxy/sorting_hat.py]: the memory usage does not need to be set explicitly?

File: files/galaxy-test/dynamic_rules/usegalaxy/destination_specifications.yaml (hunk 1 above)
Comment (gmauro) [reply files/galaxy-test/dynamic_rules/usegalaxy/destination_specifications.yaml]: Pulsar way

File: files/galaxy-test/dynamic_rules/usegalaxy/sorting_hat.py (hunk 2 above)
Comment (gmauro) [reply files/galaxy-test/dynamic_rules/usegalaxy/sorting_hat.py]: Yes, it should

File: files/galaxy-test/dynamic_rules/usegalaxy/sorting_hat.py (hunk 3 above)
Comment (gmauro) [reply files/galaxy-test/dynamic_rules/usegalaxy/sorting_hat.py]: is done in the kwargs block

**Pull request merged**